packages/core/src/client.ts

```diff
@@ -230,9 +230,6 @@ export const Client: new (opts: ClientOptions) => Client = function Client(
       onPush(result => {
         replays.set(operation.key, result);
       }),
-      onStart(() => {
-        active.set(operation.key, source);
-      }),
```

Comment:
Context on this: `onStart` only triggers _after_ the source has started, so if someone subscribes and gets a synchronous result, that result would arrive _before_ `onStart`'s callback is called. This is fixed by moving this to `make` whose inner function is called immediately.